.github/workflows/ci.yaml

```diff
@@ -68,22 +68,36 @@ jobs:
         run: |
           brew update
           brew install gitversion nuget
-          brew install abseil googletest google-benchmark
+          brew install googletest google-benchmark
 
-          if [ -x /usr/local/bin/brew ]; then
-            arch -x86_64 /usr/local/bin/brew update
-            arch -x86_64 /usr/local/bin/brew install abseil googletest google-benchmark
-          fi
+      - name: Build Abseil from source
+        run: |
+          git clone https://github.com/abseil/abseil-cpp.git
+          cd abseil-cpp
+          # Check out the revision specified in your repository.
```

Comment:
Long term we probably should just vendor this too. But I'm happy for this for now.